mcs/class/System.IO.Compression/Test/System.IO.Compression/ZipTest.cs

```diff
@@ -239,6 +239,31 @@ public void ZipCreateArchive()
 		}
 
 		[Test]
+		public void ZipEnumerateEntriesModifiedTime()
```

Comment:
OK, I think I know the issue. It's because it's using the `Now` datetime, which is the default, and since it isn't comparing seconds it makes the test pass.